### PR TITLE
chore: omit 'docfx' from docs-presubmit build

### PR DESCRIPTION
**Pull request by tseaver**
We don't have a 'docfx' session in our noxfile (see #822).

Toward #797.

### CODE REVIEW

**Comment by tseaver** [comment]
Note that I don't actually expect this PR to fix the `docs-presubmit` build, which has been failing in the `.kokoro/bulid.sh` setup before ever getting to the point of running `nox`.

File: .kokoro/build.sh
Comment (dandhlee) [inline .kokoro/build.sh:34]: `cat: /tmpfs/src/gfile/project-id.json: No such file or directory` I think it was fine before when you used `project-id.txt`?

Comment (tseaver) [reply .kokoro/build.sh]: @dandhlee That filename works for the main Kokoro build, but *not* for the `docs-presubmit` build (see [this failure](https://source.cloud.google.com/results/invocations/a3937c80-4f2f-48a0-a9cf-d6dfa090d34a/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fgoogle-auth-library-python%2Fdocs%2Fdocs-presubmit/log)).

Maybe there is something weird in the `google3` Kokoro config for this repository?  I honestly can't figure out why `docs-presubmit` is failing here.

File: owlbot.py
Comment (dandhlee) [inline owlbot.py:45]: Adding the last bit to change `@nox.session(python="3.7")` to `@nox.session(python="3.8")` for `docs(session)` on noxfile.py should get docs session to run successfully now!

Comment (tseaver) [reply owlbot.py]: I fixed that in 0c44e6a by modifying `noxfile.py` directly, because `owlbot.py` skips over it.

File: .kokoro/docker/docs/Dockerfile
Comment (dandhlee) [inline .kokoro/docker/docs/Dockerfile:73]: do we need `python2` if we don't need to run system tests?

Comment (tseaver) [reply .kokoro/docker/docs/Dockerfile:73]: This PR isn't trying to fix #832 (at least not yet).  To do that, we need either to have a `.kokoro/build-systests.sh` script, which runs only the systests, or else work out some environment variable which we use to turn them on / off in `.kokoro/build.sh`.

Either way will require adding something like `.kokoro/presubmit/system-3.8.cfg`, to either set the environment variable or trigger the alternate script, which will then need a corresponding tweak to the `google3` Kokoro configuration for this repostitory.

Comment (dandhlee) [reply .kokoro/docker/docs/Dockerfile:73]: Currently it just skips the system tests (missing 3.7 interpreter), and `docs` session runs fine. I think it's fine to move forward as-is.

**Comment by dandhlee** [comment]
Think we're almost there. Blocked by system tests but it'll be fixed soon, docs-presubmit is happy without `docfx` and runs `docs` session fine. Not sure why OwlBot isn't happy though.

**Comment by tseaver** [comment]
@dandhlee Owlbot gets its nose in a twist when the PR is rebased to remove its earlier, unwanted commits.

**Comment by dandhlee** [comment]
Hm, I thought rerunning  it would resolve itself but I guess not.

**Comment by tseaver** [comment]
@dandhlee I think we need to get a refresh-token PR merged to `master` and then here, first.

**Comment by tseaver** [comment]
Blocked on #834.

**Comment by tseaver** [comment]
@dandhlee PTAL.

**Comment by tseaver** [comment]
Added `do not merge` label pending the `google3` change to activate the `Kokoro system-3.7` build.

File: .kokoro/docker/docs/Dockerfile
Comment (tseaver) [reply .kokoro/docker/docs/Dockerfile:73]: PR #833 is based on this PR:  once we merge here, I will rebase it against `master`, and then wait to merge until the `google3` changes are in place to enable the `Kokoro system-3.7` build.

**Comment by tseaver** [comment]
Oops, the Owlbot failure is real:  the substitution I added breaks when I run synthool locally.

**Review by dandhlee — APPROVED** [review]
LGTM.

**Pull request merged**
